### DISCUSSION TITLE
keys for snr and pastro

### DISCUSSION THREAD

**Discussion by jkanner** [gwosc-tutorial/gwosc-catalog#75] [General]
The GWOSC website uses these keys:

* 'network_matched_filter_snr'
* 'p_astro'

But the current community catalog schema instead uses `pastro` and `snr`.  Why?  Should't we use the same names?

**Comment by DrKentB** [comment]
This was discussed in some detail in past meetings (I was originally pushing for these to use the same name but was talked out of it, possibly by you). I'm okay with the schema using SNR and Pastro instead of the longer names after that debate. In fact, I think Colin raised isssue with 'network_matched_filter_snr' at our last meeting. My own perspective on this is that SNR comes in too many flavors to try and nail down a name as specific as above. The difference for 'pastro' and 'p_astro' is too minor to get worked up about. 

I was asked at the time to have the code internally convert the schema key to the gwosc database field name, which I did.

**Reply by martinberoiz** [reply]
FWIW This is also how I remember it

**Reply by jkanner** [reply]
OK.  Sounds like we already discussed this and landed at the present solution.  I am OK to leave this "as-is".

**Reply by jkanner** [reply]
@DrKentB @martinberoiz   Thank you both for the quick response.

**Reply by DrKentB** [reply]
I will add that the documentation might benefit from saying that pastro is mapped to gwosc's p_astro and snr is mapped to gwosc's network_matched_filter_snr. This would allow having a key, e.g., 'mySNR' in the json file that is mapped to 'mySNR' in gwosc land, which is something that I think Colin was also wanting to be able to do. 

I guess the other thing to document is that one should not use 'p_astro' and 'network_matched_filter_snr' if they have already used 'pastro' and 'snr', because that would result in overwriting one or the other, depending on which comes first and which comes second in the json file.  (... now I'm wavering on my previous agreement - crazy stuff)
